src/js/containers/LimitOrder/withFavorite.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { connect } from "react-redux"
+import * as common from "../../utils/common";
+import limitOrderServices from "../../services/limit_order";
+import * as limitOrderActions from "../../actions/limitOrderActions";
```

Comment:
Can you sort import as action -> services -> utils

Reply:
Updated